src/test/java/duke/DeadlinesTest.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlinesTest {
+
+    @Test
+    public void deadlinesTest(){
```

Comment:
For your test files, there is another way to name your test methods following this format - `featureUnderTest_testScenario_expectedBehavior()` 

Perhaps you can follow the format for clearer understanding of each testcase. Otherwise, I think it is fine to leave it as it is as well! 😊